[main] CLI-based tools in eng directory failing to build with error CS0246

Related to https://github.com/dotnet/sdk/pull/48523

https://dev.azure.com/dnceng/internal/_build/results?buildId=2689462&view=logs&j=fa5c2909-0e29-5671-d5eb-16d6c40e0c4a&t=2a0bd486-3f67-5c59-da0c-b84d88848a4b&l=23 (internal Microsoft link)

`Scan for binaries` building failed with below message:

    /mnt/vss/_work/1/s/eng/tools/BinaryToolKit/Program.cs(61,20): error CS0246: The type or namespace name 'Command' could not be found (are you missing a using directive or an assembly reference?) [/mnt/vss/_work/1/s/eng/tools/BinaryToolKit/BinaryToolKit.csproj]
    /mnt/vss/_work/1/s/eng/tools/BinaryToolKit/Program.cs(71,42): error CS0246: The type or namespace name 'Command' could not be found (are you missing a using directive or an assembly reference?) [/mnt/vss/_work/1/s/eng/tools/BinaryToolKit/BinaryToolKit.csproj]
    /mnt/vss/_work/1/s/eng/tools/BinaryToolKit/Program.cs(12,28): error CS0246: The type or namespace name 'Argument<>' could not be found (are you missing a using directive or an assembly reference?) [/mnt/vss/_work/1/s/eng/tools/BinaryToolKit/BinaryToolKit.csproj]
    /mnt/vss/_work/1/s/eng/tools/BinaryToolKit/Program.cs(18,28): error CS0246: The type or namespace name 'Option<>' could not be found (are you missing a using directive or an assembly reference?) [/mnt/vss/_work/1/s/eng/tools/BinaryToolKit/BinaryToolKit.csproj]
    /mnt/vss/_work/1/s/eng/tools/BinaryToolKit/Program.cs(25,28): error CS0246: The type or namespace name 'Option<>' could not be found (are you missing a using directive or an assembly reference?) [/mnt/vss/_work/1/s/eng/tools/BinaryToolKit/BinaryToolKit.csproj]
    /mnt/vss/_work/1/s/eng/tools/BinaryToolKit/Program.cs(33,28): error CS0246: The type or namespace name 'Option<>' could not be found (are you missing a using directive or an assembly reference?) [/mnt/vss/_work/1/s/eng/tools/BinaryToolKit/BinaryToolKit.csproj]

This is also failing in the CreateBaselineUpdatePR project:

```
/mnt/vss/_work/1/s/eng/tools/CreateBaselineUpdatePR/Program.cs(87,20): error CS0246: The type or namespace name 'Command' could not be found (are you missing a using directive or an assembly reference?) [/mnt/vss/_work/1/s/eng/tools/CreateBaselineUpdatePR/CreateBaselineUpdatePR.csproj]
```

https://dev.azure.com/dnceng/internal/_build/results?buildId=2689642&view=results (internal link)

--- a/src/SourceBuild/content/eng/Versions.props
+++ b/src/SourceBuild/content/eng/Versions.props
@@ -28,7 +28,7 @@
     <!-- arcade dependencies -->
     <MicrosoftDotNetVersionToolsVersion>10.0.0-beta.25110.3</MicrosoftDotNetVersionToolsVersion>
     <!-- command-line-api dependencies -->
-    <SystemCommandLineVersion>2.0.0-beta4.24126.1</SystemCommandLineVersion>
+    <SystemCommandLineVersion>2.0.0-beta5.25208.1</SystemCommandLineVersion>
     <!-- msbuild dependencies -->
     <MicrosoftBuildVersion>17.12.6</MicrosoftBuildVersion>
     <!-- nuget dependencies -->

